### PR TITLE
#125 included MIT license in package.json

### PR DESCRIPTION
**Pull request by bajibpathan**
I have included MIT license information package.json file. Please review and advise.

Thank you,

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -13,5 +13,6 @@
     "install:tests": "cd ./workspaces/tests && pnpm install",
     "install:calendar-widgets": "cd ./workspaces/calendar-widgets && pnpm install",
     "new": "pnpm run install:tests && pnpm run install:calendar-widgets && pnpm run install:docs"
-  }
+  },
+  "license": "MIT"
```
Comment (9mbs) [inline package.json:17]: This should be relocated to the `package.json` file within the `workspaces/calendar-widgets/` directory. That way, when we publish the package to NPM, the license will be found in the distributable code as well 🙂

**Comment by 9mbs** [comment]
Nice work @bajibpathan! Just 1 comment

File: package.json (hunk 1 above)
Comment (bajibpathan) [reply package.json]: Thanks for the guidance. As suggested, I've relocated the license information to workspaces/calendar-widgets/package.json.

**Comment by 9mbs** [comment]
This looks perfect. Merging this into the `main` branch now. Thank you @bajibpathan!!

**Pull request merged**